src/olympia/amo/tests/test_views.py

```diff
@@ -433,7 +433,7 @@ def test_allow_mozilla_collections(self):
                              settings.TASK_USER_ID)
         response = self.client.get('/robots.txt')
         assert response.status_code == 200
-        assert 'Allow: {}'.format(url) in response.content
+        assert b'Allow: {}'.format(url) in response.content
```

Comment:
I'm sure bytes don't have a `.format()` method in python 3. I think you should decode `response.content` instead

Reply:
Yeah, that's right. I really should ban the habit of using `b'...' in response.content`.

Reply:
https://github.com/mozilla/addons-server/pull/10598/commits/5d120d5615b5e994375eb638789fe082975484ac